aries-site/src/pages/components/button.mdx

```diff
@@ -381,3 +379,19 @@ For button groups on screen widths less than or equal to 576px (Grommet "small")
     <ButtonBadGroupPreview />
   </Example>
 </BestPracticeGroup>
+
+## Accessibility
+### A11y guidance
+The following guidance should be considered for each instance of the component and is the responsibility of the implementer. WCAG criteria provided directly by the Grommet component are summarized under [WCAG compliance](#wcag-compliance).
+
+**Note**: WCAG rules marked as conditional indicate that the implementer must take additional actions to ensure full compliance.
+
+If pressing a Button results in a new URL, or the resultant UI makes sense as "a new browser tab," that is a link `<a>`. Everything else is a `<button>`.
+
+The distinction is important to screen reader users to know what's going to happen next. Will I navigate somewhere or will something happen on the page? So it's important to choose the right HTML element for the job.
+
+If using an `<a>` tag, see the [WCAG rules for the Anchor component](https://design-system.hpe.design/components/anchor#wcag-compliance).
```

Comment:
We don't want to open a new browser tab...

```suggestion
If using an `<a>` tag, see the [WCAG rules for the Anchor component](/components/anchor#wcag-compliance).
```

Reply:
Fixed